source/widgets/ProjectStatus/js/octo-status.js

```diff
@@ -98,15 +110,20 @@ function OctopusStatusWidget() {
                                         }
                                     } else {
                                         $projectH2.text('');
-                                        $environmentH3.text("No deployment found for " + settings.projectName + " to " + settings.environmentName);
+                                        if (settings.spaceName) {
+                                            $environmentH3Span.text("No deployment found for " + settings.projectName + " to " + settings.environmentName + " in " + settings.spaceName + " space.");
```

Comment:
A bit of a grammar note
Should we _either_ remove `space` at the end of the sentence, or add a `the` in front of the space name...

Reply:
I've added "the".